packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -159,8 +159,8 @@ const sharedPaddingStyles = {
   paddingRight: '16px',
   selectors: {
     [`@media screen and (min-width: ${ScreenWidthMinLarge}px)`]: {
-      paddingLeft: '32px',
-      paddingRight: '32px'
+      paddingLeft: '15px',
```

Comment:
Why are this changes to `Panel` inside the default styles in `office-ui-fabric-react` package? Shouldn't these be in the `fluent-theme` package?

Reply:
You're right about that, sorry! Working on updating now.